### PR TITLE
feat(che): check che capacity status before creating workspace

### PR DESCRIPTION
**Pull request by corinnekrych**
WIP
fixes https://github.com/openshiftio/openshift.io/issues/3426

@ibuziuk I'm not sure how to test full capacity, each time i asked for status i get `clusterFull: false`

```
curl https://api.prod-preview.openshift.io/api/codebases/che/state -H "Authorization: Bearer $TOKEN"
{"clusterFull":false,"multiTenant":true,"running":true}
```
could you test with full capacity reached?

### CODE REVIEW

File: src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts [hunk 1]
```diff
@@ -51,28 +50,41 @@ export class CodebasesItemActionsComponent implements OnDestroy, OnInit {
...
           });
-        }
-      }, error => {
-        this.workspaceBusy = false;
-        this.handleError('Failed to create workspace', NotificationType.DANGER);
-      }));
+      } else {
+        // display error message
+        this.notifications.message({
+          message: `Workspace capacity is reached, new workspace cannot be started.`,
```
Comment (ibuziuk) [inline src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts]: @corinnekrych could you please change the message to `OpenShift Online cluster is currently out of capacity, workspace cannot be started`  ?

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2870-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2870-fabric8-ui.badger.fabric8.io

**Comment by rhopp** [comment]
I've tried the testing version and it prevents user from creating new workspace. But shouldn't it also prevent from starting already created workspace, when the cluster is currently full?

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2870-3 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2870-fabric8-ui.badger.fabric8.io

**Pull request merged**